pkg/ebpf/tracer.go

```diff
@@ -92,6 +92,22 @@ func NewTracer(config *Config) (*Tracer, error) {
 		return nil, fmt.Errorf("could not load bpf module: %s", err)
 	}
 
+	// Enable kernel probes used for offset guessing.
+	// TODO: Disable them once offsets have been figured out.
+	for _, probeName := range offsetGuessProbes(config) {
+		if err := m.EnableKprobe(string(probeName), maxActive); err != nil {
+			return nil, fmt.Errorf(
+				"could not enable kprobe(%s) used for offset guessing: %s",
+				probeName,
+				err,
+			)
+		}
+	}
+
+	if err := guess(m, config); err != nil {
+		return nil, fmt.Errorf("failed to init module: error guessing offsets: %v", err)
+	}
```

Comment:
Thoughts on adding an `INFO` log if offsets WERE figured out?

Reply:
You mean logging each field name and the associated offset?

Reply:
No, just a log to indicate successful progress - `INFO: network structure offsets guessing complete...` or something along those lines.

Totally fine if you think we should exclude it though.

Reply:
ah ok. totally fine for me. I'll add it.